src/main/external-resources/UMS.conf

```diff
@@ -454,7 +460,7 @@ fully_played_action =
 # Default: ""
 fully_played_output_directory =
 
-# Minimum watched play time
+# Minimum watched play time (non-GUI)
 # -------------------------
```

Comment:
@Sami32 I haven't looked, but if we should have such a marking we should do it on all "non-GUI" settings. Is that implemented?